src/modules/FileLocksmith/FileLocksmithUI/FileLocksmithXAML/Views/MainPage.xaml

```diff
@@ -38,6 +40,7 @@
     <Grid RowSpacing="8">
         <Grid.RowDefinitions>
             <RowDefinition x:Name="ButtonsRow" Height="Auto" />
+            <RowDefinition x:Name="InfoBarRow" Height="Auto" />
```

Comment:
Should these two be `x:Name` or `Name`?

Reply:
They are interchangeable according to docs.
https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/x-name-attribute